src/pages/takathatt.tsx

```diff
@@ -89,4 +87,4 @@ const mapStateToProps = (state: RootState) => ({
 export default connect(
     mapStateToProps,
     dispatchProps
-)(withTranslation('common')(withRouter(ContributePage)));
+)(withRouter(ContributePage));
```

Comment:
Why you removing? :O

Reply:
There are no translations on that page. So it is not needed. :)